docs/topics/whatsnew1620.md

```diff
@@ -668,7 +668,7 @@ If you are publishing a `klib` and want to use only relative paths of source fil
 <tab title="Kotlin" group-key="kotlin">
 
 ```kotlin
-tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile::class).configureEach {
+tasks.withType(org.jetbrains.kotlin.gradle.dsl.KotlinCompile::class).configureEach {
```

Comment:
If it was a mistake — we should fix it.
But if this is how it was used in Kotlin 1.6.20 — we shouldn't fix it.

Reply:
Change is also fine for Kotlin 1.6.20